pkg/tfgen/generate_schema.go

```diff
@@ -312,6 +312,7 @@ func (g *schemaGenerator) genPackageSpec(pack *pkg) (pschema.PackageSpec, error)
 	if pi := g.info.Python; pi != nil {
 		spec.Language["python"] = rawMessage(map[string]interface{}{
 			"requires": pi.Requires,
+			"readme":   readme,
 		})
 	}
```

Comment:
You should do something similar to the `nodeData` approach above, where the readme is set outside the check for a non-`nil` `g.info.Python`. Not all packages will specify Python-specific information, but all packages will need the Python-specific README, since we will generate Python code regardless.